src/components/confirm/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component, Event, EventEmitter, h, Host, Prop, State, VNode } from "@stencil/core"
+import { Color, Data, Fill } from "../../model"
+
+@Component({
+	tag: "smoothly-button-confirm",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyButtonConfirm {
+	timer?: NodeJS.Timeout
+	@Prop() color?: Color
+	@Prop({ reflect: true, mutable: true }) name: string
+	@Prop() doubleClickTime = 0.2
+	@Prop({ reflect: true }) expand?: "block" | "full"
+	@Prop() fill?: Fill
+	@Prop({ reflect: true }) disabled = false
+	@Prop() shape?: "rounded"
+	@Prop() type: "link" | "button" = "button"
+	@Prop() size: "small" | "large" | "icon" | "flexible"
+	@State() clicked: number | undefined
```

Comment:
This is not obvious from the name that this is a timeStamp. I first thought it kept track of number of clicks.

Maybe call it something like `clickTimeStamp`?